crazyclock.ino

```diff
@@ -203,8 +214,8 @@ void updateDisplayedTime() {
   // just to compare real time and the fake one
   Serial.print("RTC Time:");
   DateTime fromRtc = RTClib::now();
-  FakeTime rtc = FakeTime(fromRtc.hour(), fromRtc.minute(), fromRtc.second());
-  rtc.formatTime(formattedTimeBuffer);
+  FakeTime real = FakeTime(fromRtc.hour(), fromRtc.minute(), fromRtc.second());
```

Comment:
We had a conflict here. I named this variable `rtc`, but that name [was already used](https://github.com/The-Coobaz/crazyclock/blob/main/crazyclock.ino#L50) by DS3231 real time clock.

Reply:
Was it the reason of non-working clock?